homeassistant/components/cover/gogogate2.py

```diff
@@ -0,0 +1,100 @@
+"""
+Support for Gogogate2 Garage Doors.
+
+For more details about this platform, please refer to the documentation
+https://home-assistant.io/components/cover.gogogate2/
+"""
+import logging
+
+import voluptuous as vol
+
+from homeassistant.components.cover import CoverDevice
+from homeassistant.const import (
+    CONF_USERNAME, CONF_PASSWORD, STATE_CLOSED, CONF_IP_ADDRESS, CONF_API_KEY)
+import homeassistant.helpers.config_validation as cv
+
+REQUIREMENTS = ['pygogogate2==0.0.2']
+
+_LOGGER = logging.getLogger(__name__)
+
+DEFAULT_NAME = 'gogogate2'
+
+NOTIFICATION_ID = 'gogogate2_notification'
+NOTIFICATION_TITLE = 'Gogogate2 Cover Setup'
+
+COVER_SCHEMA = vol.Schema({
+    vol.Required(CONF_USERNAME): cv.string,
+    vol.Required(CONF_PASSWORD): cv.string,
+    vol.Required(CONF_IP_ADDRESS): cv.string,
+    vol.Optional(CONF_API_KEY): cv.string,
```

Comment:
Please add `CONF_NAME` to allow the user to overwrite the name of the device. Set the default to `DEFAULT_NAME`.

Reply:
Added DEFAULT_NAME